src/style/styles.scss

```diff
@@ -1,4 +1,32 @@
 @import '_reset';
-@import 'FiveDay';
+@import '_variables';
+@import '_homepage';
+@import '_header-style';
 @import '_settings';
-@import 'header-style';
+@import '_five-day';
+
+html {
+  box-sizing: border-box;
+}
+
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+
+body {
+  background-color: $color-main-background;
+  font-family: 'Open Sans';
```

Comment:
You should probably consider providing a fallback font stack.

https://developer.mozilla.org/en-US/docs/Web/CSS/font-family

So you might do something like this:

```css
font-family: 'Open Sans', 'Helvetica', 'Arial', sans-serif;
```